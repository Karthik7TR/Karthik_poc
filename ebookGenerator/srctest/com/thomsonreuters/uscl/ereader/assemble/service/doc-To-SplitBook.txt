FrontMatterTitle|FrontMatterTitle.html|1
Copyright|Copyright.html|1
ResearchAssistance|ResearchAssistance.html|1
WestlawNext|WestlawNext.html|1
I90AD73D079DE11DDB771B1D8BA9725E3|N0B264080BFE211D8AE2C9667069B36F5.html|1|img1,img2
I8A56C8C0849E11E4AAC3EC14B1EA3C19|NBAC58782AFF711D8803AE0632FEDDFBF.html|2
I9B684A80849E11E4AAC3EC14B1EA3C19|NC0B797F0AFF711D8803AE0632FEDDFBF.html|2
I84DDB8E0849E11E4AAC3EC14B1EA3C19|NBAA41CD1AFF711D8803AE0632FEDDFBF.html|2
I65B09CD0849E11E4AAC3EC14B1EA3C19|NBD05DA91AFF711D8803AE0632FEDDFBF.html|2
I71A9F360849E11E4AAC3EC14B1EA3C19|NB427F4D1AFF711D8803AE0632FEDDFBF.html|2
I6F513650849E11E4AAC3EC14B1EA3C19|NB70942D0AFF711D8803AE0632FEDDFBF.html|2
IA7EAA730849E11E4AAC3EC14B1EA3C19|NBAC73530AFF711D8803AE0632FEDDFBF.html|2
I630049E0849E11E4AAC3EC14B1EA3C19|NB4CA2840AFF711D8803AE0632FEDDFBF.html|2
I73DC15A0849E11E4AAC3EC14B1EA3C19|NB6E9D3F0AFF711D8803AE0632FEDDFBF.html|2
I92EDE180849E11E4AAC3EC14B1EA3C19|NB715ED00AFF711D8803AE0632FEDDFBF.html|2
I97F284B0849E11E4AAC3EC14B1EA3C19|NB6974721AFF711D8803AE0632FEDDFBF.html|2
I6CCE5C00849E11E4AAC3EC14B1EA3C19|NBD5448B0AFF711D8803AE0632FEDDFBF.html|2
I91C23ED006D511DD9038A00A056A2B70|N2FCB3E80BFE311D8ABD6CC93AF995F92.html|4|imm
IABAA2030849E11E4AAC3EC14B1EA3C19|NB6C77EE1AFF711D8803AE0632FEDDFBF.html|4
I81882540849E11E4AAC3EC14B1EA3C19|NB9047690AFF711D8803AE0632FEDDFBF.html|4
I8EDB8D90849E11E4AAC3EC14B1EA3C19|NC0864EC0AFF711D8803AE0632FEDDFBF.html|4
I7B959FA0849E11E4AAC3EC14B1EA3C19|NB76322A0AFF711D8803AE0632FEDDFBF.html|4
I87BA24E0849E11E4AAC3EC14B1EA3C19|NBCA480B0AFF711D8803AE0632FEDDFBF.html|4
IA31E2B00849E11E4AAC3EC14B1EA3C19|NBAB9EEC1AFF711D8803AE0632FEDDFBF.html|4
I76646E30849E11E4AAC3EC14B1EA3C19|NB867C161AFF711D8803AE0632FEDDFBF.html|5
I695588F0849E11E4AAC3EC14B1EA3C19|NC15E5F40AFF711D8803AE0632FEDDFBF.html|5
