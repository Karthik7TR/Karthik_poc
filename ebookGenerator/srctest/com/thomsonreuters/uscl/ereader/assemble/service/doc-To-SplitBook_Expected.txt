FrontMatterTitle|FrontMatterTitle.html|1
Copyright|Copyright.html|1
ResearchAssistance|ResearchAssistance.html|1
Westlaw|Westlaw.html|1
N0B264080BFE211D8AE2C9667069B36F5|N0B264080BFE211D8AE2C9667069B36F5.html|1|img1.html,img2.html,
NBAC58782AFF711D8803AE0632FEDDFBF|NBAC58782AFF711D8803AE0632FEDDFBF.html|2
NC0B797F0AFF711D8803AE0632FEDDFBF|NC0B797F0AFF711D8803AE0632FEDDFBF.html|2
NBAA41CD1AFF711D8803AE0632FEDDFBF|NBAA41CD1AFF711D8803AE0632FEDDFBF.html|2
NBD05DA91AFF711D8803AE0632FEDDFBF|NBD05DA91AFF711D8803AE0632FEDDFBF.html|2
NB427F4D1AFF711D8803AE0632FEDDFBF|NB427F4D1AFF711D8803AE0632FEDDFBF.html|2
NB70942D0AFF711D8803AE0632FEDDFBF|NB70942D0AFF711D8803AE0632FEDDFBF.html|2
NBAC73530AFF711D8803AE0632FEDDFBF|NBAC73530AFF711D8803AE0632FEDDFBF.html|2
NB4CA2840AFF711D8803AE0632FEDDFBF|NB4CA2840AFF711D8803AE0632FEDDFBF.html|2
NB6E9D3F0AFF711D8803AE0632FEDDFBF|NB6E9D3F0AFF711D8803AE0632FEDDFBF.html|2
NB715ED00AFF711D8803AE0632FEDDFBF|NB715ED00AFF711D8803AE0632FEDDFBF.html|2
NB6974721AFF711D8803AE0632FEDDFBF|NB6974721AFF711D8803AE0632FEDDFBF.html|2
NBD5448B0AFF711D8803AE0632FEDDFBF|NBD5448B0AFF711D8803AE0632FEDDFBF.html|2
N2FCB3E80BFE311D8ABD6CC93AF995F92|N2FCB3E80BFE311D8ABD6CC93AF995F92.html|4
NB6C77EE1AFF711D8803AE0632FEDDFBF|NB6C77EE1AFF711D8803AE0632FEDDFBF.html|4
NB9047690AFF711D8803AE0632FEDDFBF|NB9047690AFF711D8803AE0632FEDDFBF.html|4
NC0864EC0AFF711D8803AE0632FEDDFBF|NC0864EC0AFF711D8803AE0632FEDDFBF.html|4
NB76322A0AFF711D8803AE0632FEDDFBF|NB76322A0AFF711D8803AE0632FEDDFBF.html|4
NBCA480B0AFF711D8803AE0632FEDDFBF|NBCA480B0AFF711D8803AE0632FEDDFBF.html|4
NBAB9EEC1AFF711D8803AE0632FEDDFBF|NBAB9EEC1AFF711D8803AE0632FEDDFBF.html|4|img5.html,
NB867C161AFF711D8803AE0632FEDDFBF|NB867C161AFF711D8803AE0632FEDDFBF.html|5
NC15E5F40AFF711D8803AE0632FEDDFBF|NC15E5F40AFF711D8803AE0632FEDDFBF.html|5
